src/qforte/utils/exponentiate.py

```diff
@@ -0,0 +1,74 @@
+"""
+Functions for exponentiation of qubit operator terms (circuits)
+"""
+
+import qforte
+import numpy
+
+
+def exponentiate_single_term(param, term):
+    """
+    A function which returns an exponentiated version of a single (param * term)
+    from a parameterized qubit state preparation circuit.
+
+    :param term: the term to be exponentiated
+    """
+
+    # TODO: look into multiplication by 1.0j? (Nick)
+    # TODO: This code is very similar to that used in PyQuil,
```

Comment:
@fevangelista, as noted in the TODO, this chunk of code is similar to PyQuill's, what is the best way to reference it (or should it be rewritten?)

Reply:
I think this is a standard implementation of the exponential as found in the literature. I have some suggestions for improving it a bit.